tests/python/gaia-ui-tests/gaiatest/apps/ftu/app.py

```diff
@@ -334,9 +334,20 @@ def a11y_click_next_to_welcome_browser_section(self):
             Wait(self.marionette).until(expected.element_present(
                 *self._section_welcome_browser_locator))))
 
+    @property
+    def is_statistics_enabled(self):
+        return self.marionette.find_element(
+            *self._statistic_checkbox_locator).is_selected()
+
     def tap_statistics_checkbox(self):
         self.marionette.find_element(*self._enable_statistic_checkbox_locator).tap()
 
+    def toggle_statistics(self):
```

Comment:
I don't think it's either-or. Enable, Disable, and Toggle are all valid semantics. The difference is in what's guaranteed. Enable should enable no matter the previous setting, ditto Disable. Toggle means literally, whatever it is now, make it the other thing. Sometimes that's what you want.

In this case I wanted Toggle, because doing so let me use a simple control variable in the test instead of having to edit Enable/Disable if the default changes again. It's also consistent with handling in Settings (home of many checkboxes :) which is what I based many consistency decisions on.

I suppose I could have done Set_Shared_Data(value) to unconditionally set to a given value, but in this case I really did want to say "flip it and make sure it flips," not so much "set it here."

It's a fine point, and I'm hoping we can go with this (given new naming) for now. It's something to discuss again in terms of pattern best practices, for sure.